Makefile

```diff
@@ -16,7 +16,7 @@ else
   DICTFLAGS = -c -p
 endif
 
-PACKAGE = OscProb
+PACKAGE = OscProb2
```

Comment:
Why this change?

Reply:
Oops... I wasn't trying to make that change.  I'll change it back.  It was that way on my computer because I was trying to keep this modified package separate from the original OscProb package.